app/routes/settings/email-preferences.js

```diff
@@ -7,19 +7,6 @@ export default Route.extend({
     return this.l10n.t('Email Preferences');
   },
   model() {
-    return [{
-      name             : 'Techtoma',
-      role             : 'Organiser',
-      isNewPaper       : true,
-      isChangeSchedule : true,
-      isNewEvent       : true
-    },
-    {
-      name             : 'Open Summit 2017',
-      role             : 'Organiser',
-      isNewPaper       : false,
-      isChangeSchedule : false,
-      isNewEvent       : false
-    }];
+    return this.get('authManager.currentUser').get('emailNotifications');
```

Comment:
Make sure you include event & event name in this request itself. So that a new request isn't being made for event. 

- https://flask-rest-jsonapi.readthedocs.io/en/latest/include_related_objects.html
- https://flask-rest-jsonapi.readthedocs.io/en/latest/sparse_fieldsets.html